src/modules/75-cd/components/PipelineSteps/CloudFormation/CloudFormationCreateStack/CloudFormationCreateStack.tsx

```diff
@@ -0,0 +1,262 @@
+/*
+ * Copyright 2021 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import React, { forwardRef } from 'react'
+import * as Yup from 'yup'
+import { isEmpty, map, set } from 'lodash-es'
+import { IconName, MultiTypeInputType, getMultiTypeFromValue } from '@harness/uicore'
+import { yupToFormErrors, FormikErrors } from 'formik'
+import { StepViewType, StepProps, ValidateInputSetProps } from '@pipeline/components/AbstractSteps/Step'
+import type { ExecutionElementConfig } from 'services/cd-ng'
+import { StepType } from '@pipeline/components/PipelineSteps/PipelineStepInterface'
+import { PipelineStep } from '@pipeline/components/PipelineSteps/PipelineStep'
+import { getDurationValidationSchema } from '@common/components/MultiTypeDuration/MultiTypeDuration'
+import {} from '../CloudFormationInterfaces'
+import { CloudFormationCreateStack } from './CloudFormationCreateStackRef'
+
+const CloudFormationCreateStackWithRef = forwardRef(CloudFormationCreateStack)
+
+export type ProvisionersOptions = 'CLOUD_FORMATION'
+export interface CloudFormationData {
+  provisioner: ExecutionElementConfig
+  originalProvisioner?: Partial<ExecutionElementConfig>
+  provisionerEnabled: boolean
+  provisionerSnippetLoading?: boolean
+  selectedProvisioner?: ProvisionersOptions
+}
+export interface CloudFormationProps {
+  initialValues: CloudFormationData
+  template?: CloudFormationData
+  path?: string
+  readonly?: boolean
+  stepViewType?: StepViewType
+  onUpdate?: (data: CloudFormationData) => void
+  onChange?: (data: CloudFormationData) => void
+  allowableTypes: MultiTypeInputType[]
+}
+
+export class CFCreateStack extends PipelineStep<any> {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
export class CFCreateStack extends PipelineStep<unknown> {
```